[2024-03-29T17:01:53.852+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_templating.t2 manual__2024-03-29T17:01:50.584301+00:00 [queued]>
[2024-03-29T17:01:53.913+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_templating.t2 manual__2024-03-29T17:01:50.584301+00:00 [queued]>
[2024-03-29T17:01:53.917+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T17:01:54.012+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): t2> on 2024-03-29 17:01:50.584301+00:00
[2024-03-29T17:01:54.049+0000] {standard_task_runner.py:60} INFO - Started process 3241 to run task
[2024-03-29T17:01:54.120+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'demo_templating', 't2', 'manual__2024-03-29T17:01:50.584301+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/demo_templating.py', '--cfg-path', '/tmp/tmp9hc1c6sn']
[2024-03-29T17:01:54.171+0000] {standard_task_runner.py:88} INFO - Job 47: Subtask t2
[2024-03-29T17:01:54.496+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T17:01:55.143+0000] {task_command.py:423} INFO - Running <TaskInstance: demo_templating.t2 manual__2024-03-29T17:01:50.584301+00:00 [running]> on host c4cb237119ff
[2024-03-29T17:01:56.744+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='demo_templating' AIRFLOW_CTX_TASK_ID='t2' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T17:01:50.584301+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-29T17:01:50.584301+00:00'
[2024-03-29T17:01:56.781+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-29T17:01:56.788+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 2']
[2024-03-29T17:01:56.940+0000] {subprocess.py:86} INFO - Output:
[2024-03-29T17:01:57.161+0000] {subprocess.py:93} INFO - 2
[2024-03-29T17:01:57.221+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-29T17:01:57.904+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=demo_templating, task_id=t2, execution_date=20240329T170150, start_date=20240329T170153, end_date=20240329T170157
[2024-03-29T17:01:58.589+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T17:02:01.235+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
